src/test/java/org/intellij/sdk/codesync/clients/ClientUtilsTest.kt

```diff
@@ -0,0 +1,20 @@
+package org.intellij.sdk.codesync.clients
+
+import org.intellij.sdk.codesync.Constants
+import org.intellij.sdk.codesync.clients.ClientUtils
+import org.junit.jupiter.api.AfterEach
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Test
+import kotlin.test.assertEquals
+import kotlin.test.assertNotNull
+import kotlin.test.assertNull
+
+class ClientUtilsTest {
+
+    @Test
+    fun sendPost(){
+        val correctAPI = String.format("%s/users?&source=%s&v=%s", "https://api.codesync.com/v1", "intellij", "unknown")
```

Comment:
@syedgulahmed Kindly point this to localhost. Tests should not be run against producntion. 
Please also open a ticket to mock this server request later. Thanks

Reply:
Okay,

Reply:
Yep, that sounds good.

Reply:
Actually, the right way for tests is to mock the server call. That way test won't have external dependency and will run in every environment. You can look at some mocking examples from https://github.com/codesyncapp/codesync-intellij/blob/master/src/test/java/org/intellij/sdk/codesync/files/ConfigFileTest.kt#L73

Reply:
> Actually, the right way for tests is to mock the server call. That way test won't have external dependency and will run in every environment. You can look at some mocking examples from https://github.com/codesyncapp/codesync-intellij/blob/master/src/test/java/org/intellij/sdk/codesync/files/ConfigFileTest.kt#L73

Yeah, I have created ticket for that to add mocking for this test case. This is just temporary.